R/pam-tidiers.R

```diff
@@ -0,0 +1,90 @@
+' Tidying methods for pam objects
+#' 
+#' These methods summarize the results of Partitioning Around Medoids (PAM) clustering into three
+#' tidy forms. `tidy` describes the medoid and size of each cluster,
+#' `augment` adds the cluster assignments to the original data, and
+#' `glance` summarizes the average silhouette with of the clustering.
+#'
+#' @templateVar class pam
+#' @template title_desc_tidy
+#' 
+#' @param x An `pam` object returned from [cluster::pam()]
+#' @param col.names Dimension names. Defaults to the names of the variables in x.  Set to NULL to get names `x1, x2, ...`.
```

Comment:
What are the "dimension names"? This is just the columns in the data frame, right? Perhaps clarify this here.